jni/include/org_opensearch_knn_jni_FaissService.h

```diff
@@ -72,15 +72,15 @@ JNIEXPORT void JNICALL Java_org_opensearch_knn_jni_FaissService_setSharedIndexSt
  * Signature: (J[FI[I)[Lorg/opensearch/knn/index/query/KNNQueryResult;
  */
 JNIEXPORT jobjectArray JNICALL Java_org_opensearch_knn_jni_FaissService_queryIndex
-  (JNIEnv *, jclass, jlong, jfloatArray, jint, jintArray);
+  (JNIEnv *, jclass, jlong, jfloatArray, jint, jobject, jintArray);
```

Comment:
as you are changing the signature of the function change this too.

```
Signature: (J[FI[I)[Lorg/opensearch/knn/index/query/KNNQueryResult;
```

Reply:
Missed it. will do